examples/bridge-app/linux/main.cpp

```diff
@@ -0,0 +1,190 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <platform/CHIPDeviceLayer.h>
+#include <platform/PlatformManager.h>
+
+#include "af.h"
+#include "gen/attribute-id.h"
+#include "gen/cluster-id.h"
+#include <app/chip-zcl-zpro-codec.h>
+#include <app/util/af-types.h>
+#include <app/util/attribute-storage.h>
+#include <app/util/util.h>
+#include <core/CHIPError.h>
+#include <setup_payload/QRCodeSetupPayloadGenerator.h>
+#include <setup_payload/SetupPayload.h>
+#include <support/CHIPMem.h>
+#include <support/RandUtils.h>
+
+#include "LightingManager.h"
+#include "Options.h"
+#include "Server.h"
+
+#include <cassert>
+#include <iostream>
+
+using namespace chip;
+using namespace chip::Inet;
+using namespace chip::Transport;
+using namespace chip::DeviceLayer;
+
+void emberAfPostAttributeChangeCallback(EndpointId endpoint, ClusterId clusterId, AttributeId attributeId, uint8_t mask,
```

Comment:
you might consider tracking lighting state per-endpoint in the LightingMgr. Seems like right now all endpoints will track to a single internal state.

Reply:
> Good start. I think it will be soon be important to track per-endpoint state.

Yes, I think we can tackle this aspect in the follow-up patches.